engine/class_modules/sc_warrior.cpp

```diff
@@ -3934,6 +3937,14 @@ struct avatar_t : public warrior_spell_t
     warrior_spell_t::execute();
 
     p()->buff.avatar->trigger();
+
+    if ( p()->azerite.bastion_of_might.enabled() )
+    {
+      p()->buff.bastion_of_might->trigger();
+
+      action_t* bastion_of_might_ip_active = p()->active.bastion_of_might_ip;
+      bastion_of_might_ip_active->execute( );
```

Comment:
```suggestion
      p() -> active.bastion_of_might_ip -> execute( );
```
You can just execute it right away without assigning it to a new bastion_of_might_ip_variable